tokens/src/lib.rs

```diff
@@ -216,6 +216,12 @@ pub mod module {
 		/// Handler to burn or transfer account's dust
 		type OnDust: OnDust<Self::AccountId, Self::CurrencyId, Self::Balance>;
 
+		/// Handler for when an account was created
+		type OnNewTokenAccount: Happened<(Self::AccountId, Self::CurrencyId)>;
+
+		/// Handler for when an account was created
+		type OnKilledTokenAccount: Happened<(Self::AccountId, Self::CurrencyId)>;
```

Comment:
I think we can have a default implementation for those two new trait. maybe use `Nothing`?

Reply:
There should be a default implementation for `Happened` for `()` which does nothing?

Reply:
Or are you talking about default implementations for traits? I thought that wasn't in stable Rust, yet?

Reply:
`()` would be ok.